components/ca-form.js

```diff
@@ -333,6 +333,8 @@ define([
 
                 // erase current form
                 clearElement(this.form);
+                this.setAttribute('create-url', '');
```

Comment:
Regardless, should not be setting attributes directly - should use the property setter createUrl('') and updateUrl('')